mypyc/genops.py

```diff
@@ -130,6 +131,7 @@ def flush_errors(self) -> None:
             print(error)
 
 
+@strict_optional_set(True)  # Turn on strict optional for any type manipulations we do
```

Comment:
Something is weird here, since this fails type check.

Reply:
I think the stubs for callable contextmanagers are just wrong